index.html

```diff
@@ -8,29 +8,38 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-
     <main>
         <header>
             <nav>
-                <a href="#" id="brand">Emre Karayaman</a>
-                <img src="img/burger-icon.png" alt="menu-icon" id="burger-menu-icon">
+                <a href="#headline" id="brand">Emre Karayaman</a>
+                <ul class="wide wide-nav">
+                    <li class="nav-links">
+                        <a href="#portfolio">Portfolio</a>
+                    </li>
+                    <li class="nav-links">
+                        <a href="#about">About</a>
+                    </li>
+                    <li class="nav-links">
+                        <a href="#contact">Contact</a>
+                    </li>
+                </ul>
+                <img class="mobile" src="img/burger-icon.png" alt="menu-icon" id="burger-menu-icon">
             </nav>
         </header>
```

Comment:
- [ ] Good job so far your project design is impressive :+1: But please try to make your top navigation similar to the Figma template by removing the `box-shadow` and your menu option color needs to be the same as defined on Figma because it s important to stick the design as much as possible  :smile: 
You can see the difference below :arrow_down: 

#### Your navigation design
![Screenshot from 2022-08-24 10-05-37](https://user-images.githubusercontent.com/6615394/186391638-ecbfea36-53ca-4f18-89b5-f25ea1d67f55.png)

#### Figma design
![Screenshot from 2022-08-24 10-02-55](https://user-images.githubusercontent.com/6615394/186391642-f2b0b3c7-3d02-4ab7-90d0-0efda39a1683.png)